htdocs/don/class/don.class.php

```diff
@@ -113,6 +113,18 @@ class Don extends CommonObject
 	public $phone;
 	public $phone_mobile;
 
+	public $country_olddata;
```

Comment:
No need for this. You can remove the line 
`$this->country_olddata    = $obj->country_olddata; // deprecated`